.yapfignore

```diff
@@ -0,0 +1,2 @@
+splunk_eventgen/lib/concurrent
```

Comment:
Can we use pip to install the 3rd party lib instead of putting all the code into our code repo?

Reply:
I have a story to track this.

Reply:
Ok, go ahead and remove this file once that's done then.